packages/backend/src/tokens/jwt/signJwt.ts

```diff
@@ -0,0 +1,45 @@
+import runtime from '../../runtime';
+import { base64url } from '../../util/rfc4648';
+import { getCryptoAlgorithm, importKey } from './algorithms';
+
+export interface SignJwtOptions {
+  algorithm?: string;
+  header?: Record<string, unknown>;
+}
+
+function encodeJwtData(value: unknown): string {
+  const stringified = JSON.stringify(value);
+  const encoder = new TextEncoder();
+  const encoded = encoder.encode(stringified);
+  return base64url.stringify(encoded, { pad: false });
+}
+
+export async function __unstable__signJwt(
```

Comment:
☁️ I would suggest we expose it as `signJwt` and add a JSDoc to indicate that this function will be used internally.
Since this method does not contain Clerk-specific logic, why not expose it without the `__unstable` prefix?
cc: @nikosdouvlis , @SokratisVidros

Reply:
@dimkl the naming convention helps with also preventing confusion on the user's side on whether they should use this function in their flow or not as well. We could add a JSDoc annotation regardless though.

Reply:
I think we should not use the approach of exposing internal methods as `__unstable`, since it does not keep this as internal and it pollutes the public API of our package with "weird" prefixed exports. I know that `__unstable` prefixed exports are used in some parts of our packages, but I think that we should only use them as a dirty way to support a case that we don't have another to way to support at the moment (eg `__unstable_options`).

If it's something users should not use, why add it in `@clerk/backend`?
How possible is it that the `signJwt` signature will change with a breaking change?

I am okay with `__unstable_signJwt` but my personal suggestion is:
- export `signJwt`
- add JSDoc to indicate that this is internal
- (optional) add docs for `signJwt` to explain that this should not be used to generate clerk JWTs, that is being used internally and that it could be used as a helper to generate a signature for other cases expect creating a Clerk JWT (eg sign a payload in the customer application server).

Reply:
Went with this approach 😄  Also added js docs about the signature of the function!